server/embedded/src/main/resources/log4j2.xml

```diff
@@ -173,19 +173,19 @@ account managers so that we can coordinate edits with other customized copies of
         </Root>
 
         <!-- category for server side script messages -->
-        <Logger name="org.labkey.api.script.ScriptService.Console" level="info">
+        <Logger name="org.labkey.api.script.ScriptService.Console" additivity="false" level="info">
```

Comment:
Not sure, but I think we probably want logging to the console as well, which suggests removing additivity.